app/code/Magento/Catalog/Controller/Adminhtml/Product/Attribute/Save.php

```diff
@@ -124,9 +141,18 @@ public function execute()
                 }
             }
 
+            /* @var $model ModelAttribute */
+            $model = $this->attributeFactory->create();
+
             $attributeId = $this->getRequest()->getParam('attribute_id');
-            $attributeCode = $this->getRequest()->getParam('attribute_code')
-                ?: $this->generateCode($this->getRequest()->getParam('frontend_label')[0]);
+            if($attributeId){
```

Comment:
Please remove ` @codingStandardsIgnoreFile` in the top of file, such formatting violates PSR-2 but it is not checked until there is an annotation.